### ISSUE TITLE
Profil Pythonisty?

### ISSUE THREAD

**Issue by honzajavorek** [pyvec/python.cz#139]
- na webu python.cz je už teď jakýsi seznam firem a jednotlivců
- firmy možná budou moct mít profil, ze začátku hlavně za to, když napíší success story
- jednotlivci by mohli mít profil, kde by byl aspoň nějaký rozcestník na linkedin, twitter, web, navolnenoze profil, atd. - dá se to snadno vygenerovat a pomohlo by to dohledatelnosti pythonistů u nás a utužení komunity

Má to smysl? Nemá to smysl? Chce to někdo?

**Comment by petrjoachim** [comment]
Co tak mít jen seznam PSF členů z české komunity?

**Comment by encukou** [comment]
Sem by se mělo z py.cz zmigrovat [HonzaSvec](http://www.py.cz/PyCz/HonzaSvec) :)

**Comment by frenzymadness** [comment]
Já jsem rozhodně pro. Například teď po sprintu by se mi to hodilo, když jsem hledal kontakty na některé jeho účastníky a případně jejich konta na soc. sítích.

**Comment by frenzymadness** [comment]
Klidně jen něco jednoduchého ve stylu "Tým pražských PyLadies" na http://pyladies.cz/praha_info/

**Comment by honzajavorek** [comment]
Jo, to vypada dobre, takhle by to klidne stacilo. Muze se to generovat z
nejakyho yamlu, stejne jako uz nektery veci na python.cz, aby se to dalo
snadno pridavat pres PR.

**Comment by messa** [comment]
Jestli by to mělo fungovat i pro hledání/nabízení koučování (což by mohlo), tak inspirací může být https://pehapkari.cz/mentori/

**Comment by honzajavorek** [comment]
Tož rýsujou se dvě věci. Senzam lidí nějak aktivních v organizaci komunity, tzn. lidí hlavně z Pyvec Slacku, GitHubu, apod., kteří se potřebují znát a vědět kdo má jakou funkci z organizačních důvodů. Tento seznam bych směřoval na stránky Pyvec.org nebo do Pyvec Guide návodů. Mám to rozpracováno.

Pak je tu veřejný seznam jakýchkoliv lidí, kteří se chtějí prezentovat jako Pythonisti. Pro to nyní nevidím velké využití a ani touhu od těch jakýchkoliv lidí se takto prezentovat, ale klidně mě opravte.

Následně je tady ta dobrá myšlenka co se týče mentorování. To by opravdu mohl být nějaký seznam na Python.cz v sekci pro začátečníky, kde by měli možnost kontaktovat konkrétní lidi z různých koutů republiky. Přidat větu o tom, že takto může přijít i na Pyvo/nePyvo/Pyworking Sessions, nebo že existují PyLadies Remote, nalinkovat [přednášku od Anny](https://github.com/honzajavorek/become-mentor), atd.

**Comment by messa** [comment]
> Pak je tu veřejný seznam jakýchkoliv lidí, kteří se chtějí prezentovat jako Pythonisti. Pro to nyní nevidím velké využití a ani touhu od těch jakýchkoliv lidí se takto prezentovat, ale klidně mě opravte.

Kdo má extra touhu (typicky freelanceři), už je na https://python.cz/prace/ Zbytek je ve FB anebo nějaké LinkedIn skupině. To mi přijde dostatečné, resp. taky nevidím žádné využití pro cokoliv dalšího v tomto směru.

---

Pyladies Remote ani neznám, dík za info, [doplním](https://github.com/messa/what-next) :)

Ale pro holky už je toho dost, na FB se ozývají spíš začátečníci - kluci.

Z online skupin pro začátečníky pak existuje ještě [Pyworking Praha](https://www.facebook.com/groups/PyWorkingPraha/), [Učíme Python](https://www.facebook.com/groups/ucimepython/), pak tu jsou ještě Czechitas...

**Comment by frenzymadness** [comment]
>> Pak je tu veřejný seznam jakýchkoliv lidí, kteří se chtějí prezentovat jako Pythonisti. Pro to nyní nevidím velké využití a ani touhu od těch jakýchkoliv lidí se takto prezentovat, ale klidně mě opravte.

> Kdo má extra touhu (typicky freelanceři), už je na https://python.cz/prace/ Zbytek je ve FB anebo nějaké LinkedIn skupině. To mi přijde dostatečné, resp. taky nevidím žádné využití pro cokoliv dalšího v tomto směru.

Já sám o sobě také ne, protože ty lidi nemám potřebu hledat. Na druhou stranu, když si představím situaci, že se něco čerstvě učím a zapojuji se do komunity, raděj bych měl někde seznam lidí, kteří o sobě dobrovolně avizují co dělají a že jsou ochotni pomoci. Tipovat náhodně zapojené lidi do FB skupin nebo si vybírat náhodně koho oslovím podle jeho linkedin profilu mi nepříjde nejlepší cesta a navíc to introvertní zájemce o pomoc odradí ještě dříve než to zkusí.

**Comment by messa** [comment]
> raděj bych měl někde seznam lidí, kteří o sobě dobrovolně avizují co dělají a že jsou ochotni pomoci.

K tomu by byl ten seznam koučů, o kterém psal @honzajavorek.

**Comment by honzajavorek** [comment]
PyLadies Remote jsou fajn zmínit i když je toho pro holky už dost, protože
je to inspirace i pro “takové to domácí mentorování”. Jinými slovy, lidi si
všimnou, že to jde i na dálku, když se chce.
